pkgs/top-level/all-packages.nix

```diff
@@ -5496,6 +5496,8 @@ with pkgs;
 
   g2o = libsForQt5.callPackage ../development/libraries/g2o { };
 
+  gatus = callPackage ../servers/monitoring/gatus { };
+
```

Comment:
```suggestion
```

New top-level packages should go to `pkgs/by-name` https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md